tests/store/test_sqlalchemy_store.py

```diff
@@ -324,37 +323,7 @@ def test_create_run(self):
         self.assertEqual(actual.info.source_version, expected["source_version"])
         self.assertEqual(actual.info.entry_point_name, expected["entry_point_name"])
         self.assertEqual(actual.info.start_time, expected["start_time"])
-        self.assertEqual(len(actual.data.tags), 3)
-
-        name_tag = models.SqlTag(key=MLFLOW_RUN_NAME, value='booyya').to_mlflow_entity()
-        self.assertListEqual(actual.data.tags, tags + [name_tag])
-
-    def test_create_run_with_parent_id(self):
```

Comment:
This test is making sure that the parent pointer is not lost. if we are moving this logic to fluent--prolly makes sense to add a relevant testcase in `test_fluent.py`

Reply:
I'm not sure what you mean by the 'parent pointer' here. If you're referring to a top level attribute containing parent run ID (as opposed to a tag), I don't believe there is one.

There is already a test in `test_fluent` called `test_start_run_with_parent` that checks that the relevant tag gets passed to the client - can toy have a look and see if that satisfies your requirements?